index.bs

```diff
@@ -1409,6 +1431,8 @@ Gets the {{DOMException}} object passed to {{SensorErrorEventInit}}.
 
     1.  If |sensor_instance|.{{[[state]]}} is "activated",
         1.  Let |readings| be the [=latest reading=] of |sensor_instance|'s related [=platform sensor=].
+        1.  If the [=extension specification=] defines a [=local coordinate system=] for |sensor_instance|,
+            1. Remap |readings| values to the [=local coordinate system=].
```

Comment:
Maybe make reference to coordinate system transformation (reference frame). There should be plenty of books, articles.

Reply:
Thanks for the link!